app/dashboard/settings/page.tsx

```diff
@@ -0,0 +1,39 @@
+"use client"
+import { Input } from '@/components/ui/input'
```

Comment:
I deleted this page because the user cant edit anything.
Maybe in the future we open this page up, but I think it may confuse the user at this point.